src/Drupal/DrupalExtension/Context/MinkContext.php

```diff
@@ -98,7 +98,7 @@ public function beforeJavascriptStep($event)
         }
         $text = $event->getStep()->getText();
         if (preg_match('/(follow|press|click|submit)/i', $text)) {
-            $this->iWaitForAjaxToFinish();
+            $this->iWaitForAjaxToFinish($event);
```

Comment:
This variable doesn't appear to be defined anywhere...

Reply:
`public function beforeJavascriptStep($event)`

It is passed to the method :)

Reply:
haha, oops.